### PR TITLE
Luca's interview from Mainmatter's Rust interview series

### PR DESCRIPTION
**Pull request by LukeMathWalker**
As discussed on Discord, this is the entry for the first podcast episode from the Rust interview series curated by Mainmatter.

A few questions/notes:

- I have put tomorrow as date, but obviously feel free to move it according to your overall publishing schedule
- I have followed the convention w.r.t. file naming, but I assume that's up to you @jonhoo 
- The audio file I have sent you @jonhoo doesn't include Rustacean Station's theme as intro. All the timings would have to be shifted by its duration if that's prepended to the file.

### CODE REVIEW

File: _episodes/interviews/2023-08-10-luca-palmieri.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Adopting Rust: present and future of the Rust web ecosystem, with Luca Palmieri"
+date: 2023-08-10T10:00:00Z
```
Comment (jonhoo) [inline _episodes/interviews/2023-08-10-luca-palmieri.md:3]: I'll update this tomorrow when I actually take the steps to release the episode :+1:

File: _episodes/interviews/2023-08-10-luca-palmieri.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+### Credits
+
+Intro Theme: [Aerocity](https://twitter.com/AerocityMusic)
+
+Audio Editing: [Mainmatter](https://mainmatter.com)
+
+Hosting Infrastructure: [Jon Gjengset](https://twitter.com/jonhoo/)
+
+Show Notes: [Plangora](https://twitter.com/plangora)
```
Comment (jonhoo) [inline _episodes/interviews/2023-08-10-luca-palmieri.md:41]: You should set this to yourself, given you wrote these :p

File: _episodes/interviews/2023-08-10-luca-palmieri.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+Marco Otto-Witte discusses with [Luca Palmieri](https://lpalmieri.com) the present and future of the Rust web ecosystem. 
+
+### Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
```
Comment (jonhoo) [inline _episodes/interviews/2023-08-10-luca-palmieri.md:16]: @bstrie We should get Mastodon into the template.

File: _episodes/interviews/2023-08-10-luca-palmieri.md (hunk 2 above)
Comment (LukeMathWalker) [reply _episodes/interviews/2023-08-10-luca-palmieri.md:41]: I wasn't sure what we meant by "Show Notes" 😂

Comment (LukeMathWalker) [reply _episodes/interviews/2023-08-10-luca-palmieri.md]: OK, fixed in b8aae8b4d105915854fbced35ffc5cb3e2dc19b4

**Comment by jonhoo** [comment]
Sorry, this completely ran me by! Will post it tomorrow instead (since we're posting #223 today). Would you mind updating the file name (including the audio file URL) to have 2023-08-11 instead?

**Comment by LukeMathWalker** [comment]
No problem at alls—done in [d1fbe68](https://github.com/rustacean-station/rustacean-station.org/pull/224/commits/d1fbe6890db86123415e9b9056a5fc2cbc82fe61)!

File: _episodes/interviews/2023-08-12-luca-palmieri.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Adopting Rust: present and future of the Rust web ecosystem, with Luca Palmieri"
+date: 2023-08-12T10:00:00Z
```
Comment (jonhoo) [inline _episodes/interviews/2023-08-12-luca-palmieri.md:3]:
```suggestion
date: 2023-08-12T13:57:00Z
```

**Comment by jonhoo** [comment]
Aaand it's out!

**Pull request merged**